### PR TITLE
Local Kubernetes script update

### PR DESCRIPTION
**Pull request by stevenGravy**
The current script for getting the webapi url results in a couple errors:

- url includes `\` char

```bash
EXTERNAL_IP=$(kubectl get service teleport-cluster -o jsonpath='{ .status.loadBalancer.ingress[0].ip }')
curl --insecure https://$\{EXTERNAL_IP\?\}:443/webapi/ping
curl: (6) Could not resolve host: $EXTERNAL_IP
```

- example json example not excluded from copy.  commented out now

### CODE REVIEW

File: docs/pages/try-out-teleport/local-kubernetes.mdx [hunk 1]
```diff
@@ -83,8 +83,8 @@ Verify that Teleport is running.
 
 ```code
 $ kubectl get pods
-NAME                               READY   STATUS    RESTARTS   AGE
-teleport-cluster-b9dc6c68b-d75fb   1/1     Running   0          46s
+# NAME                               READY   STATUS    RESTARTS   AGE
```
Comment (zmb3) [inline docs/pages/try-out-teleport/local-kubernetes.mdx:86]: I don't like commenting out the output - I find it confusing.

The goal of a code snippet should either be:

1. Make it easy to copy a long or error-prone command
2. Demonstrate what the output of a typical command looks like

As you've seen here, it's hard to do both well.

In this case, I don't think we need to optimize for copying `kubectl get pods`. It's a command that kube users are very familiar with, and hard to get wrong.

If you really want to solve for both items above, we could do separate blocks. One for the command, and then another ```text block that shows what the output might look like.

File: docs/pages/try-out-teleport/local-kubernetes.mdx [hunk 2]
```diff
@@ -125,8 +125,9 @@ HTTP API is working:
 
 ```code
 $ EXTERNAL_IP=$(kubectl get service teleport-cluster -o jsonpath='{ .status.loadBalancer.ingress[0].ip }')
-$ curl --insecure https://${EXTERNAL_IP?}:443/webapi/ping
-{"auth":{"type":"local","second_factor":"otp","preferred_local_mfa":"otp","has_motd":false},"proxy":{"kube":{"enabled":true,"listen_addr":"0.0.0.0:3026"},"ssh":{"listen_addr":"[::]:3023","tunnel_listen_addr":"0.0.0.0:3024","public_addr":"teleport-cluster:443"},"db":{"mysql_listen_addr":"0.0.0.0:3036"},"tls_routing_enabled":false},"server_version":"8.2.0","min_client_version":"7.0.0"}%
+$ TELEPORT_PING_URL="https://${EXTERNAL_IP?}:443/webapi/ping"
+$ curl --insecure $TELEPORT_PING_URL
+# {"auth":{"type":"local","second_factor":"otp","preferred_local_mfa":"otp","has_motd":false},"proxy":{"kube":{"enabled":true,"listen_addr":"0.0.0.0:3026"},"ssh":{"listen_addr":"[::]:3023","tunnel_listen_addr":"0.0.0.0:3024","public_addr":"teleport-cluster:443"},"db":{"mysql_listen_addr":"0.0.0.0:3036"},"tls_routing_enabled":false},"server_version":"8.2.0","min_client_version":"7.0.0"}%
```
Comment (zmb3) [inline docs/pages/try-out-teleport/local-kubernetes.mdx:130]: This long line is hard to read - maybe pipe it through jq?

File: docs/pages/try-out-teleport/local-kubernetes.mdx [hunk 3]
```diff
@@ -125,8 +125,9 @@ HTTP API is working:
 
 ```code
 $ EXTERNAL_IP=$(kubectl get service teleport-cluster -o jsonpath='{ .status.loadBalancer.ingress[0].ip }')
-$ curl --insecure https://${EXTERNAL_IP?}:443/webapi/ping
-{"auth":{"type":"local","second_factor":"otp","preferred_local_mfa":"otp","has_motd":false},"proxy":{"kube":{"enabled":true,"listen_addr":"0.0.0.0:3026"},"ssh":{"listen_addr":"[::]:3023","tunnel_listen_addr":"0.0.0.0:3024","public_addr":"teleport-cluster:443"},"db":{"mysql_listen_addr":"0.0.0.0:3036"},"tls_routing_enabled":false},"server_version":"8.2.0","min_client_version":"7.0.0"}%
+$ TELEPORT_PING_URL="https://${EXTERNAL_IP?}:443/webapi/ping"
```
Comment (alexfornuto) [inline docs/pages/try-out-teleport/local-kubernetes.mdx:128]: Could this and the line above be combined? If we're not going to use the `EXTERNAL_IP` var except to create the `TELEPORT_PING_URL` var, why not add the logic from the former into the latter? I don't have a working cluster to test this, but something like:

```suggestion
$ TELEPORT_PING_URL="https://$(kubectl get service teleport-cluster -o jsonpath='{ .status.loadBalancer.ingress[0].ip }'):443/webapi/ping"
```

**Review by alexfornuto — APPROVED** [review]
One suggestion to reduce complexity, otherwise 👍

File: docs/pages/try-out-teleport/local-kubernetes.mdx (hunk 2 above)
Comment (alexfornuto) [reply docs/pages/try-out-teleport/local-kubernetes.mdx:130]: I generally agree, but we have to be careful about our assumptions of the reader's config; they may not have `jq` installed.

Comment (zmb3) [reply docs/pages/try-out-teleport/local-kubernetes.mdx:130]: We could omit the `| jq` from the example and display the formatted JSON in the docs page anyway. Not _exactly_ what they'd see if they ran it, but probably an improvement over this long line of JSON.

File: docs/pages/try-out-teleport/local-kubernetes.mdx (hunk 3 above)
Comment (alexfornuto) [reply docs/pages/try-out-teleport/local-kubernetes.mdx:128]: Never mind, I see we use `EXTERNAL_IP` later on. I'll add a comment here to explain.

File: docs/pages/try-out-teleport/local-kubernetes.mdx (hunk 2 above)
Comment (alexfornuto) [reply docs/pages/try-out-teleport/local-kubernetes.mdx]: I adjusted it to include the pipe but also explain it in the copy.

File: docs/pages/try-out-teleport/local-kubernetes.mdx (hunk 1 above)
Comment (alexfornuto) [reply docs/pages/try-out-teleport/local-kubernetes.mdx]: I've adjusted the use of comments for output in this PR.

File: docs/pages/try-out-teleport/local-kubernetes.mdx [hunk 4]
```diff
@@ -121,12 +121,39 @@ teleport-cluster   LoadBalancer   10.107.218.212   127.0.0.1     443:32143/TCP,3
 
 The Proxy Service enables you to manage your cluster via an HTTP API. Assign the
 external IP of the load balancer to an environment variable and verify that the
-HTTP API is working:
+HTTP API is working. The example below uses [jq](https://stedolan.github.io/jq/) to format the API return, but you can omit it if not installed:
```
Comment (zmb3) [inline docs/pages/try-out-teleport/local-kubernetes.mdx:124]:
```suggestion
HTTP API is working. The example below uses [jq](https://stedolan.github.io/jq/)
to format the API response, but you can omit it if not installed:
```

**Review by zmb3 — APPROVED** [review]

**Comment by alexfornuto** [comment]
@stevenGravy should we manually backport this to v10 & v9?

**Comment by stevenGravy** [comment]
Yes I will do that later thanks!

**Pull request merged**